server/routes/setup_terraform_environment.py

```
@@ -95,76 +95,39 @@ def write_tfvars(data: Dict[str, Any], workdir: str) -> str:
-                os.environ["TF_VAR_tenant_id"] = str(tenant_id)
-                os.environ["TF_VAR_client_id"] = str(client_id)
-                logger.info(f"Azure Terraform environment configured for subscription: {cached_subscription_id} (cached)")
-                return True, cached_subscription_id, None
-        except Exception as e:
-            logger.debug(f"Cached Azure setup unavailable; falling back to direct setup: {e}")
+        from chat.backend.agent.tools.auth import setup_azure_environment_cached
 
-        # Get Azure credentials
-        from utils.auth.cloud_auth import generate_azure_access_token
+        selected_subscription_id = get_selected_project_id()
+        ok, subscription_id, auth_method, cached_env = setup_azure_environment_cached(
```

Ruff (0.15.10)

[warning] Unpacked variable `auth_method` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)